src/LinuxTracing/KernelTracepoints.h

```diff
@@ -10,37 +10,37 @@
 // Format is based on the content of the event's format file:
 // /sys/kernel/debug/tracing/events/<category>/<name>/format
 
-struct __attribute__((__packed__)) tracepoint_common {
+struct __attribute__((__packed__)) TracepointCommon {
```

Comment:
As you said, the reason was to be closer to the man page.

But also, it kept a difference between these, that mimic the format in the ring buffers (view on the raw memory), and the `PerfEvent` hierarchy, which is data read from the ring buffers and stored in a way that's convenient for us. For me this distinction was always helpful.